src/components/20-molecules/stepper/ui.test.js

```diff
@@ -0,0 +1,222 @@
+import { ClientFunction, Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+
+fixture('Stepper - basic functionality').page(
+  `${host}/iframe.html?id=components-stepper--stepper`
+);
+
+const TAG = 'axa-stepper';
+const CLASS = '.m-stepper';
+
+test('should render stepper', async t => {
+  const $axaElem = Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+
+  const $axaElemShadow = Selector(
+    () => document.querySelector(TAG).shadowRoot,
+    { dependencies: { TAG } }
+  );
+
+  const $axaElemShadowEl = $axaElemShadow.find(CLASS);
+  await t.expect($axaElemShadowEl.exists).ok();
+});
+
+test('should render correct amount of steps', async t => {
+  const amountOfSteps = ClientFunction(
+    () => {
+      const shadow = document.querySelector(TAG).shadowRoot;
+      return shadow.querySelectorAll('.js-stepper__step').length;
+    },
+    { dependencies: { TAG } }
+  );
+
+  await t.expect(await amountOfSteps()).eql(4);
+});
+
+test('should display correct step number', async t => {
+  const getStepNumber = ClientFunction(
+    index => {
+      const shadow = document.querySelector(TAG).shadowRoot;
+      const step = shadow.querySelectorAll('.js-stepper__circle')[index];
+
+      return step.textContent;
+    },
+    { dependencies: { TAG } }
+  );
+
+  await t.expect(await getStepNumber(0)).eql('2');
+  await t.expect(await getStepNumber(1)).eql('3');
```

Comment:
DRY violation - could you use an array of expected outcomes (const EXPECTED = [2,3,4]) and a loop?